bc_TrackingMenu.lua

```diff
@@ -47,6 +47,18 @@ bcTM_TrackingAbilityToCreatureType = {
 	[BINDING_NAME_BCTM_BINDING_TRACK_UNDEAD] = BINDING_NAME_BCTM_BINDING_TYPE_UNDEAD,
 }
 
+bcTM_TrackingAbilityToTexture = {
+	[BINDING_NAME_BCTM_BINDING_SENSE_DEMONS] = "Interface\\Icons\\Spell_Shadow_Metamorphosis",
+	[BINDING_NAME_BCTM_BINDING_SENSE_UNDEAD] = "Interface\\Icons\\Spell_Holy_SenseUndead",
+	[BINDING_NAME_BCTM_BINDING_TRACK_BEASTS] = "Interface\\Icons\\Ability_Tracking",
+	[BINDING_NAME_BCTM_BINDING_TRACK_DEMONS] = "Interface\\Icons\\Spell_Shadow_SummonFelHunter",
+	[BINDING_NAME_BCTM_BINDING_TRACK_DRAGONKIN] = "Interface\\Icons\\INV_Misc_Head_Dragon_01",
+	[BINDING_NAME_BCTM_BINDING_TRACK_ELEMENTALS] = "Interface\\Icons\\Spell_Frost_SummonWaterElemental",
+	[BINDING_NAME_BCTM_BINDING_TRACK_GIANTS] = "Interface\\Icons\\Ability_Racial_Avatar",
+	[BINDING_NAME_BCTM_BINDING_TRACK_HUMANOIDS] = "Interface\\Icons\\Spell_Holy_PrayerOfHealing",

```

Comment:
There is one minor issue: Druids use `Ability_Tracking` icon for Track Humanoids. Unlike hunters, this is their only tracking ability, so I assume it shouldn't be a big deal.